docs/getting_started.md

```diff
@@ -0,0 +1,164 @@
+
+=== "Windows"
+
+    To install `docker` on Windows you can either:
+
+    - install "Docker Desktop on Windows": [https://docs.docker.com/desktop/install/windows-install/](https://docs.docker.com/desktop/install/windows-install/) or
+    - consider using [Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/install)
+    to get a Windows supported Linux installation. You can then install `docker` using the Linux instructions
+
+    If you install "Docker Desktop on Windows" you will also install `docker compose` automatically.
+
+=== "MacOS"
+
+    Install Docker Desktop on Mac: [https://docs.docker.com/desktop/install/mac-install/](https://docs.docker.com/desktop/install/mac-install/).
+    If you install "Docker Desktop on Mac" you will also install `docker compose` automatically.
+
+??? warning "Linux is the only supported OS"
+
+    We test and deploy on Linux and ensure that our
+    deployment instructions work on Linux systems.
+
+    We also try to provide docs and help for different architectures,
+    but as a small team with limited resources we won't be able to 
+    help you debug Operating System specific problems. 
+
+Because we rely on some modern features of these
+tools, please make sure you have at least the following
+versions on your machine:
+
+- `docker` engine: [v20.10.24](https://docs.docker.com/engine/release-notes/20.10/) or greater
+```bash
+docker --version
+```
+- `docker compose`: [v2.0.0](https://github.com/docker/compose/releases/tag/v2.0.0) or above
+```bash
+docker compose version
+```
+### Get the neurobagel launch recipe
+
+The [`neurobagel/recipes` repository](https://github.com/neurobagel/recipes) 
+on GitHub contains our official
+docker compose setup recipe. 
+
+1. Clone the github repository to your machine and navigate to it
+```bash
+git clone https://github.com/neurobagel/recipes.git
+cd recipes
+```
+2. Make copies of the template config files
+```bash
+cp template.env .env
+cp local_nb_nodes.template.json local_nb_nodes.json
+```
+3. Change `NB_API_QUERY_URL` in the `.env` file 
+
+    You **must** replace the placeholder value for `NB_API_QUERY_URL`in the `.env` file
+    
+    ```bash
+    NB_API_QUERY_URL=http://XX.XX.XX.XX
+    ```
+    with the address of the machine you are going to deploy Neurobagel on.
+
+    - If you are deploying Neurobagel **for yourself** or just to try things out locally, 
+   you can use: `NB_API_QUERY_URL=http://localhost:8080` 
+    where `:8080` is the default port for the federation API.
+    - If you are deploying Neurobagel on a server for other users, 
+   you **must** use the IP (and Port) or URL that your users will access the server with. 
+   
+!!! info
+
+    This is the minimal configuration you need to make before you can launch Neurobagel.
+    In most cases, and especially when you are deploying Neurobagel for other users,
+    you will have to make additional configurations. 
```

Comment:
This PR is not about changing the defaults. In my opinion, `full_stack` is the best profile for "getting started" because it gets you from zero to "everything runs" in 3 steps. Again: once a user has gone through this docs page they will invariably have to go more in depth in the other docs pages. The idea here is not to be exhaustive / complete but to show people how quickly they can get going. 

see e.g. https://fastapi.tiangolo.com/#example is just a piece of code with zero context.

Reply:
re your suggested change: I'd say we leave it at "must change" to indicate the minimal config they just ran is not meant to be production ready.